packit_service/worker/jobs.py

```diff
@@ -107,22 +107,36 @@ def get_config_for_handler_kls(
     :param package_config: we pick the JobConfig from this package_config instance
     :return: JobConfig
     """
+    matching_jobs = []
```

Comment:
This variable is used ~10 lines bellow for the first time. How about declaring it just before that for-loop?

Reply:
I don't see much value in moving this one. This is the one used across the whole function and is returned and e.g. `matching_job_types` and `matching_required_job_types` are created before the loop where they are used. I treat those a bit differently.